test/unittests/execute_call_test.cpp

```diff
@@ -479,8 +481,9 @@ TEST(execute_call, imported_function_from_another_module)
     const auto func_idx = fizzy::find_exported_function(*module1, "sub");
     ASSERT_TRUE(func_idx.has_value());
 
-    auto sub = [&instance1, func_idx](Instance&, const Value* args, int depth) -> ExecutionResult {
-        return fizzy::execute(*instance1, *func_idx, args, depth);
+    auto sub = [&instance1, func_idx](Instance&, const Value* args, ThreadContext& ctx) {
+        // ThreadContext is bumped here.
```

Comment:
I was a bit confused by this comment. I think you mean at this point it was already bumped by the caller?

Reply:
Probably should be "is not bumped". And it should be according to the current semantic.